### PR TITLE
Fbe/load script improvements

### PR DESCRIPTION
**Pull request by fbeutin-ledger**
## Description

Add empty .bss section enforcer in loader scripts for plugin compilation
Add empty .data section enforcer in loader scripts for all targets

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Tests
- [ ] Documentation
- [ ] Other (for changes that might not fit in any category)

### CODE REVIEW

File: Makefile.rules_generic
Comment (tdejoigny-ledger) [inline Makefile.rules_generic:1]: @fbeutin-ledger question : Can we keep the existing link files (only with a conditional link instruction for plugins) ? 
--> makefile.rules_generic won't be modified
--> 4 extra linkfiles not to maintain

File: Makefile.rules_generic [hunk 1]
```diff
@@ -89,7 +89,14 @@ $(OBJ_DIR)/%.o: %.S $(BUILD_DEPENDENCIES) prepare
 	$(L)$(call as_cmdline,$(INCLUDES_PATH), $(DEFINES),$<,$@)
 
 ifeq ($(SCRIPT_LD),)
-SCRIPT_LD:=$(BOLOS_SDK)/target/$(TARGET)/script.ld
+ifeq ($(IS_PLUGIN),)
+LDFLAGS += -L$(BOLOS_SDK)/target/$(TARGET)
+SCRIPT_LD := $(BOLOS_SDK)/target/$(TARGET)/script.ld
+else
+LDFLAGS += -L$(BOLOS_SDK)/target/$(TARGET)
```
Comment (yhql) [inline Makefile.rules_generic:96]: This line is identical in both branches

File: Makefile.rules_generic
Comment (fbeutin-ledger) [reply Makefile.rules_generic:1]: Not possible natively on link scripts. Adding a new precompiled layer on it is very heavy

Comment (tdejoigny-ledger) [reply Makefile.rules_generic:1]: ok

**Review by apaillier-ledger — APPROVED** [review]

**Pull request merged**